provider/resource_keycloak_openid_client_test.go

```diff
@@ -444,6 +444,21 @@ func testAccCheckKeycloakOpenidClientExistsWithCorrectProtocol(resourceName stri
 	}
 }
 
+func testAccCheckKeycloakOpenidClientExistsWithCorrectLifespan(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		client, err := getOpenidClientFromState(s, resourceName)
+		if err != nil {
+			return err
+		}
+
+		if client.Attributes.AccessTokenLifespan != "1800" {
+			return fmt.Errorf("expected openid client to have openid-connect protocol, but got %s", client.Protocol)
```

Comment:
Could you fix this error message?  it doesn't look like this applies here

Reply:
Sorry copy and fail